lib/lcevc/lcevc_dil.js

```diff
@@ -0,0 +1,281 @@
+/*! @license
+ * Shaka Player
+ * Copyright 2016 Google LLC
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+goog.provide('shaka.lcevc.Dil');
+goog.require('shaka.log');
+goog.require('shaka.util.Dom');
+
+/**
+ * @export
+ * @summary
+ * lcevcDil provides all the operations related to the enhancement and rendering
+ * of LCEVC enabled streams and on to a canvas.
```

Comment:
Readers have no idea why this is called "DIL".  Can we either explain that or give it a clearer name?

Reply:
We have added more documentation to this and also the details about what DIL stands for.